pkg/testutil/fakes_test.go

```diff
@@ -392,6 +394,55 @@ func TestSimpleServiceLister(t *testing.T) {
 	assert.Equal(t, "test", service.Namespace)
 }
 
+func TestSimpleServiceAccountLister(t *testing.T) {
```

Comment:
I think you need one of those WHEN headers

Reply:
fixed